app/views/periodic_expenses/index.slim

```diff
@@ -5,15 +5,36 @@
 - else
 
   table
```

Comment:
This looks like the exact same table as in the expenses. Is it? Will it always be?